charts/kubeai/Chart.yaml

```diff
@@ -7,7 +7,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.20.0
+version: 0.20.1
```

Comment:
Can you revert this? I handle this as part of release process.

Reply:
Done!